src/nvim/edit.c

```diff
@@ -3311,7 +3465,7 @@ static int ins_compl_bs(void)
   // Respect the 'backspace' option.
   if ((int)(p - line) - (int)compl_col < 0
       || ((int)(p - line) - (int)compl_col == 0
-          && ctrl_x_mode != CTRL_X_OMNI) || ctrl_x_mode == CTRL_X_EVAL
```

Comment:
For some reason, pressing backspace cancels `complete()` based completion, instead of simply refreshing the list of candidates. The test in test_popup.vim references changing this behavior as a todo, so I thought I'd just do a drive by change.

Reply:
is this issue still in vim? then it could make sense to break this out to a separate PR, so it is easier to backport it to vim :)

Reply:
Makes sense. I've deleted the change